zasilkovna.php

```diff
@@ -788,13 +788,14 @@ public function plgVmDisplayListFEShipment(VirtueMartCart $cart, $selected = 0,
                     $countries = $method->countries;
                 }
             }
-            if(count($countries) && !in_array($address['virtuemart_country_id'], $countries)) {
+            // intentional type unsafe comparison, handles both string (PHP < 8.1) and int (PHP >= 8.1) returned from db
+            if (count($countries) && !in_array($address['virtuemart_country_id'], $countries, false)) {
                 continue;
             }
 
             if ($isHdCarrier) {
                 $hdCarrier = $this->carrierRepository->getCarrierById($zasMethod->getHdCarrierId());
-                if (!$hdCarrier || (int)$hdCarrier->deleted === 1 || ($countryCode !== '' && $hdCarrier->country !== $countryCode)) {
+                if (!$hdCarrier || (int)$hdCarrier->deleted === 1 || ( $countryCode !== '' && $hdCarrier->country !== $countryCode)) {
```

Comment:
Změnu pouze ve whitespace bych tu nedělal, navíc je to nekonzistentní code style.